packages/storybook-vue/stories/components/telekom-brand-header-navigation/TelekomBrandHeader.stories.mdx

```diff
@@ -432,7 +434,8 @@ Set the profile menu's `logged-in` property to `true` or `false` to switch betwe
       accessibilityLabel: "open user menu: Alexander",
       closeMenuAccessibilityLabel: "close menu",
       loginSettingsLabel: "Login Settings",
-      logoutLabel: "Log out",
+      logoutLabel: "Log out",   
+      type: "standard",
```

Comment:
Remove this since the 'standard' is not a type. it fixes the build if you remove this